core/conn/Makefile

```diff
@@ -25,16 +25,17 @@ include ../macros.gmk #top level
 
 RM      = /bin/rm
 ifeq ($(SQ_BUILD_TYPE),release)
-  CLIENT_TAR	?= ../../${DISTRIBUTION_DIR}/trafodion_clients-$(TRAFODION_VER).tgz
+  CLIENT_TAR	?= ../../${DISTRIBUTION_DIR}/apache-trafodion_clients-$(TRAFODION_VER)-incubating.tar.gz
```

Comment:
Unfortunately, that variable value is not the right value for a file name. I suppose we could add yet another variable, but that does not seem necessary for packaging name in 2-3 spots.